.clj-kondo/config/modules/config.edn

```diff
@@ -321,6 +321,7 @@
           metabase.lib.metadata.protocols
           metabase.lib.normalize
           metabase.lib.options
+          metabase.lib.parse
```

Comment:
This seems reasonable to me, but I'm not sure about the modularization plans for lib, so ping @bshepherdson here to sanity check.

Reply:
I think this is the bad valley we want to avoid. We want a hundred modules with one `core` namespace each; not 20 modules with five surfaces.

So if this new namespace makes sense as part of lib, then it should be re-exported from `metabase.lib.core` and not appear in this list. On the other hand, if this is better seen as a separate thing, then it should be a separate module that consumes the library.

Reply:
(It follows that `metabase.lib` has way too many API namespaces. It should have `metabase.lib.core` and _maybe_ `metabase.lib.metadata`. We'll get there eventually.)